docs/overview.rst

```diff
@@ -19,6 +19,24 @@ Define schemas for tabular data packages
 The schemas describe the energy systems and are defined for different
 types of components in
 `facade_attrs <https://github.com/rl-institut/oemoflex/tree/dev/oemoflex/model/facade_attrs>`_.
+
+Currently the following components can be modeled:
+
+* Assymetric storage
```

Comment:
Could use automodule for that list to show all classes in the facade module, apart from the abstract class `Facade`.

Reply:
Good idea. However it only contains the following classes: 

- Source, 
- Transformer, 
- AssymetricStorage, 
- Bev, 
- ReservoirWithPump and
- ExtractionTurbine. 

Any ideas how I get the rest?

- Back pressure turbine, 
- Excess, 
- Link, 
- Load,
- Shortage,
- Storage and
- Volatile

Reply:
We leave it like this. Linking to [oemof.tabular docs](https://oemof-tabular.readthedocs.io/en/latest/usage.html#facades) is no solution so far because the components there differ from the ones in `oemoflex docs`.